packages/app/src/domain/action-max-value-getters/getWithdrawMaxValue.ts

```diff
@@ -48,5 +48,5 @@ export function getWithdrawMaxValue({ user, asset }: GetWithdrawMaxValueParams):
     }
   }
 
-  return NormalizedUnitNumber(BigNumber.min(...ceilings).dp(asset.decimals, BigNumber.ROUND_DOWN))
+  return NormalizedUnitNumber(NormalizedUnitNumber.min(...ceilings).decimalPlaces(asset.decimals, BigNumber.ROUND_DOWN))
```

Comment:
```suggestion
  returnNormalizedUnitNumber.min(...ceilings).decimalPlaces(asset.decimals, BigNumber.ROUND_DOWN)
```
Shouldn't it work like that?